README.md

```diff
@@ -1,3 +1,55 @@
 # Referite-Backend
 This is repository of backend for referite project.
 
+## Install & Run
+
+### How to install
+
+make sure that you have [python](https://www.python.org/downloads/) in your computer
+first, clone [**this repository**](https://github.com/Referite/Referite-Backend) by type this command in your terminal at your choosen path
+
+```js
+git clone https://github.com/Referite/Referite-Backend.git Referite-Backend
+```
+
+go to project directory
+
+```js
+cd Referite-Backend
+```
+
+next, you have to create file name `.env` to configuration
+
+`.env` file template looks like [sample.env](sample.env) you can modify value and copy it into `.env`  
+**Note that you may get your .env by contacted siravich.te@ku.th or preawpan.t@ku.th via email.**
+
+next, you have to create environment by typing this command
+
+```js
+python -m venv env
+```
+
+then, activate the environment using
+
+```js
+. env/bin/activate # macOS and Linux
+env\Scripts\activate.bat # Windows
+```
+
+restart the terminal, the terminal will using environment now then typing this command to install requirements
```

Comment:
Mac user don't need to restart terminal

Reply:
Explain more about Including mocking data and running local too!

Reply:
just `source env/bin/activate` and `pip install -r requirements.txt`
and about mocking data you can look example at `ioc` repository

and running local you write that pull docker and run mongo local

Reply:
why we need that all of the api using specific connection from db.py file so it won't work

Reply:
what is this mean just source env/bin/activate and pip install -r requirements.txt

Reply:
no need to restart terminal

Reply:
> So I need to add this? and it will work? Create dump folder in mongodb folder. Copy Sota folder from bson folder in the repository to new dump folder that you create in mongodb folder.

idk but I mean README

Reply:
my window need to restart, I know it readme but it will work right?

Reply:
not Sota, Sota is his repo name, idk I will investigate hai